app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+
+import requests
+
+URL = "http://api.weatherapi.com/v1/"
+
+KEY = os.environ.get("API_KEY")
+
+FILTER = "Kharkiv"
+
+METHOD = "current.json"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(URL + METHOD + f"?key={KEY}&q={FILTER}")
+    if response.status_code == 200:
+        data = response.json()
+        print(f"Current location: {data['location']['name']}\n"
+              f"Time: {data['location']['localtime']}\n"
+              f"Temperature: {data['current']['temp_c']}")
+    else:
+        print(f"{response.status_code}")
```

Comment:
it's better to return response error instead of status code

Reply:
Also I don't think you want result.png file to be added in docker image, let's ignore it